### DISCUSSION TITLE
Android: Open external links within the Mobile App

### DISCUSSION THREAD

**Discussion by Aru-Ku** [community/community#35593] [Mobile]
When clicking on any external link, it straight away opens up users' default browser app.
It'll be rather much convenient to open the link within the mobile app, just like many Android applications does (As shown in below image).
![Example Image with In-App Browser](https://user-images.githubusercontent.com/47110396/194755740-6c0cea58-e4f6-4014-80cd-9360cf987388.png)

> Some applications also has this setting to toggle this feature.

**Comment by stevepopovich** [comment]
Hi @Aru-Ku, so your ideal behavior for our app, is that anytime you click a link that would push out to the browser, we actually launch an in-app browser instead that keeps you in the GitHub app but still opens a web view?

**Reply by Aru-Ku** [reply]
Hi @stevepopovich, yes, every link opens up in default browser (as separate app), instead of in GitHub Mobile app's web-view.

**Comment by prosultan2334** [comment]
Most of the android app websites have some external links but these kind of links are difficult to open so being an administrator of website anyone can fix this issue. I have seen the sorted issue on a [website](https://apkfog.com/), which can provide external links and all those links can be accessible from mobile.
